configs/filter_terms.txt

```diff
@@ -0,0 +1,2 @@
+New York
```

Comment:
Is this just a placeholder at the moment? It would be good to include a comment to make that clear.